neuro_evolution_ctrnn/brains/continuous_time_rnn.py

```diff
@@ -118,9 +117,9 @@ def brainstep(self, ob: np.ndarray) -> Union[np.ndarray, np.generic]:
                 y_ = np.tanh(self.y)
             elif self.config.neuron_activation == "learned":
                 # value = alpha * np.tanh(self.y) + (1-alpha) * np.relu(self.y)
-                raise NotImplementedError("learned activations are not yet implemented")
+                raise NotImplementedError("Learned activations are not yet implemented.")
             else:
-                raise RuntimeError("unknown aktivation function: " + str(self.config.neuron_activation))
+                raise RuntimeError("Unknown activation function: {}.".format(self.config.neuron_activation))
```

Comment:
I don't know actually. Its just a personal preference for me at least and since we use it almost everywhere else in the code I thought i change it. If I had to, I would say its faster and more "pythonic" but that's just a guess